template.html

```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html lang="{{lang}}">
   <head>
+    <!-- content="
+    default-src 'none';
+    font-src 'none';
+    img-src 'self';
+    script-src 'self' 'sha256-QIO/JKO6nn9l5q1faq4A06sEzbRLkirNwdq3+qJzJVQ=';
+    style-src 'self';
+  " -->
```

Comment:
You forgot to uncomment this 😉 You can generate a new hash here: https://csplite.com/csp/sha/